plugins/reporters/evaluated-model/src/main/kotlin/EvaluatedModelMapper.kt

```diff
@@ -635,8 +636,11 @@ internal class EvaluatedModelMapper(private val input: ReporterInput) {
     ) {
         val pathExcludes = getPathExcludes(id, scanResult.provenance)
         val licenseFindingCurations = getLicenseFindingCurations(id, scanResult.provenance)
+        // TODO: Decide whether the ordering in the output should be independent of the order of the input and
+        //       either remove below sortedWith() call, or replace it with a proper ordering mechanism for the whole
+        //       model.
         val curatedFindings = curationsMatcher.applyAll(scanResult.summary.licenseFindings, licenseFindingCurations)
-            .mapNotNullTo(mutableSetOf()) { it.curatedFinding }
+            .mapNotNullTo(mutableSetOf()) { it.curatedFinding }.sortedWith(LicenseFinding.COMPARATOR).toSet()
```

Comment:
I propose to instead use:

    .mapNotNull { it.curatedFinding }.toSortedSet(LicenseFinding.COMPARATOR)